src/fides/api/util/consent_util.py

```diff
@@ -228,3 +247,281 @@ def get_or_create_fides_user_device_id_provided_identity(
         )
 
     return identity  # type: ignore[return-value]
+
+
+def validate_notice_data_uses(
+    privacy_notices: List[Union[PrivacyNoticeWithId, PrivacyNoticeCreation]],
+    db: Session,
+) -> None:
+    """
+    Ensures that all the provided `PrivacyNotice`s data has valid data uses.
+    Raises a 422 HTTP exception if an unknown data use is found on any `PrivacyNotice`
+    """
+    valid_data_uses = [data_use.fides_key for data_use in DataUse.query(db).all()]
+    try:
+        for privacy_notice in privacy_notices:
+            privacy_notice.validate_data_uses(valid_data_uses)
+    except ValueError as e:
+        raise HTTPException(HTTP_422_UNPROCESSABLE_ENTITY, detail=str(e))
+
+
+def ensure_unique_ids(
+    privacy_notices: List[PrivacyNoticeWithId],
+) -> None:
+    """
+    Ensures that all the provided `PrivacyNotice`s have unique IDs
+    Raises a 422 HTTP exception if there is more than one PrivacyNotice with the same ID
+    """
+    ids = set()
+    for privacy_notice in privacy_notices:
+        if privacy_notice.id not in ids:
+            ids.add(privacy_notice.id)
+        else:
+            raise HTTPException(
+                HTTP_422_UNPROCESSABLE_ENTITY,
+                detail=f"More than one provided PrivacyNotice with ID {privacy_notice.id}.",
+            )
+
+
+def create_privacy_notices_util(
+    db: Session,
+    privacy_notice_schemas: List[PrivacyNoticeCreation],
+    should_escape: bool = True,
+) -> Tuple[List[PrivacyNotice], Set[PrivacyNoticeRegion]]:
+    """Performs validation before creating Privacy Notices and Privacy Notice History records
+    and then ensures that Privacy Experiences exist for all Privacy Notices.
+    """
+    validate_notice_data_uses(privacy_notice_schemas, db)  # type: ignore[arg-type]
+
+    existing_notices = PrivacyNotice.query(db).filter(PrivacyNotice.disabled.is_(False)).all()  # type: ignore[attr-defined]
+
+    new_notices = [
+        PrivacyNotice(**privacy_notice.dict(exclude_unset=True))
+        for privacy_notice in privacy_notice_schemas
+    ]
+    check_conflicting_data_uses(new_notices, existing_notices)
+
+    created_privacy_notices: List[PrivacyNotice] = []
+    affected_regions: Set = set()
+
+    for privacy_notice in privacy_notice_schemas:
+        if should_escape:
+            # should_escape flag is for when we're creating a notice
+            # from a template. The content was already escaped in the
+            # template - we don't want to escape twice.
```

Comment:
This method was largely relocated from the original API `create_privacy_notices` endpoint.  

It is shared between the endpoint  that creates privacy notices in the API and the method that loads out of the box privacy notices on startup.  The main change is that we can toggle whether we should escape the notices or not.  If we're creating the notices from a template, the notice was already escaped in the template so we don't want to escape it twice.

Otherwise this method should be nearly identical to the body of the original create_privacy_notices function.